app/models/video.rb

```diff
@@ -136,13 +137,30 @@ def self.import(file)
       attributes = row.to_hash.slice(*keys)
 
       video = Video.new(attributes)
-      video.subject_list.add(row['subject'])
+      if not row['subject'].nil?
+        video.subject_list.add(row['subject'].strip.downcase)
+      end
 
       video.fill_missing_fields
       video.save
     end
   end
 
+  def self.import_id(video_hash)
+    if video_hash["youtube_id"].empty? or video_hash["subject"].empty?
+      raise ArgumentError, "Please fill in required fields."
+    end
+
+    keys = [ 'description', 'title', 'youtube_id']
+    attributes = video_hash.slice(*keys)
+    
+    video = Video.new(attributes)
+    video.subject_list.add(video_hash['subject'].strip.downcase)
```

Comment:
If you're keeping the subjects lowercase, we'll have to find a way to titleize them later.